apis/Google.Cloud.Vision.V1/Google.Cloud.Vision.V1/project.json

```diff
@@ -20,7 +20,7 @@
 
   "dependencies": {
     "Google.Api.CommonProtos": "1.0.0-beta02",
```

Comment:
It won't matter - but at some point before GA we should canonicalize, of course.